scripts/circle/Dockerfile

```diff
@@ -1,36 +1,64 @@
-FROM centos:centos7.3.1611
-
-RUN yum -y update && \
-    yum -y install \
-        bzip2-devel.x86_64 \
-        curl \
-        git \
-        gcc \
-        gcc-c++ \
-        gcc-gfortran \
-        gmake \
-        hdf5-devel.x86_64 \
-        numpy.x86_64 \
-        python-devel.x86_64 \
-        https://github.com/openhpc/ohpc/releases/download/v1.3.GA/ohpc-release-1.3-1.el7.x86_64.rpm
-
-RUN yum -y update && \
-    yum -y install \
-        ohpc-base \
-        openmpi-gnu7-ohpc \
-        phdf5-gnu7-openmpi-ohpc \
-        netcdf-gnu7-openmpi-ohpc
+FROM centos:centos7
 
-WORKDIR /home/adios2
+# Install extra repos
+RUN yum -y install epel-release https://github.com/openhpc/ohpc/releases/download/v1.3.GA/ohpc-release-1.3-1.el7.x86_64.rpm && \
+    yum -y upgrade
+
+# Install core dev packages
+RUN yum -y install \
+      curl file \
+      gcc gcc-c++ gcc-gfortran \
+      git make autoconf automake libtool\
+      bzip2-devel zeromq-devel hdf5-devel \
+      python-devel numpy
+
+# Install gcc7 OpenHPC packages
+RUN yum -y install \
+      lmod-ohpc gnu7-compilers-ohpc openmpi-gnu7-ohpc phdf5-gnu7-openmpi-ohpc \
+      python-numpy-gnu7-ohpc
+
```

Comment:
It might not matter much but it's probably worth doing a `yum clean all` after all packages are installed to remove all the cached metadata.  May save a few MB

Reply:
Good idea, I'll add that.